### ISSUE TITLE
How can I create my own data like face.js? 

### ISSUE THREAD

**Issue by benzispawn** [eduardolundgren/tracking.js#383]
I have a specific need for tracking different parameters in photos and videos. How can I create a data file like face.js, mouth.js, etc?

**Comment by tuxedomurat** [comment]
yes you can. google for how to train haar cascades. It will be python or matlab etc. Might have new languages added to the list recently.

**Comment by murat-aka** [comment]
you can. google for haar cascade training. it will be python or matlab etc.
